pkgs/applications/networking/browsers/brave/default.nix

```diff
@@ -161,8 +164,19 @@ stdenv.mkDerivation rec {
 
   preFixup = ''
     # Add command line args to wrapGApp.
-    gappsWrapperArgs+=(--add-flags ${lib.escapeShellArg commandLineArgs})
+    gappsWrapperArgs+=(--prefix LD_LIBRARY_PATH : ${rpath})
+    gappsWrapperArgs+=(--prefix PATH : ${binpath} )
+    ${optionalString (enableFeatures != []) ''
+      gappsWrapperArgs+=(--add-flags "--enable-features=${strings.concatStringsSep "," enableFeatures}")
+    ''}
+    ${optionalString (disableFeatures != []) ''
+      gappsWrapperArgs+=(--add-flags "--disable-features=${strings.concatStringsSep "," disableFeatures}")
+    ''}
+    gappsWrapperArgs+=(--add-flags ${escapeShellArg commandLineArgs})
     gappsWrapperArgs+=(--add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform=wayland}}")
+    ${optionalString vulkanSupport ''
+      gappsWrapperArgs+=(--prefix XDG_DATA_DIRS  : "${ addOpenGLRunpath.driverLink }/share" )
+    ''}
```

Comment:
```suggestion
    gappsWrapperArgs+=(
      --prefix LD_LIBRARY_PATH : ${rpath}
      --prefix PATH : ${binpath}
    ${optionalString (enableFeatures != []) ''
      --add-flags "--enable-features=${strings.concatStringsSep "," enableFeatures}"
    ''}
    ${optionalString (disableFeatures != []) ''
      --add-flags "--disable-features=${strings.concatStringsSep "," disableFeatures}"
    ''}
      --add-flags ${escapeShellArg commandLineArgs}
      --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform=wayland}}"
    ${optionalString vulkanSupport ''
      --prefix XDG_DATA_DIRS  : "${ addOpenGLRunpath.driverLink }/share"
    ''}
    )
```

Reply:
Also, out of curiosity why not:
```nix
preFixup = ''
  # Add command line args to wrapGApp.
  gappsWrapperArgs+=(
    --prefix LD_LIBRARY_PATH : ${rpath}
    --prefix PATH : ${binpath}
    ${optionalString (enableFeatures != []) ''
    --add-flags "--enable-features=${strings.concatStringsSep "," enableFeatures}"
    ''}
    ${optionalString (disableFeatures != []) ''
    --add-flags "--disable-features=${strings.concatStringsSep "," disableFeatures}"
    ''}
    --add-flags ${escapeShellArg commandLineArgs}
    --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform=wayland}}"
    ${optionalString vulkanSupport ''
    --prefix XDG_DATA_DIRS  : "${addOpenGLRunpath.driverLink}/share"
    ''}
  )
'';
```
is this incorrect?